test/cpp/api/modules.cpp

```diff
@@ -1294,6 +1308,12 @@ TEST_F(ModulesTest, PrettyPrintCosineSimilarity) {
       "torch::nn::CosineSimilarity(dim=0, eps=0.5)");
 }
 
+TEST_F(ModulesTest, PrettyPrintTripletMarginLoss) {
+  ASSERT_EQ(
+      c10::str(TripletMarginLoss(TripletMarginLossOptions().margin(3).p(2).eps(1e-06).swap(false))),
+      "torch::nn::TripletMarginLoss(margin=3, p=2, eps=1e-06, swap=false)");
```

Comment:
Is it supposed to also print `reduction`? I guess we don't have a nice string representation of the reduction enum at the moment, and please feel free to not print it for now. :D

Reply:
@yf225 , no this won't be printing `reduction` here.

Reply:
@yf225 , Thanks a lot for reviewing it. I have learnt a lot from it, gained confidence and got familiar with the pytorch codebase.
Will surely make the mentioned changes and fix the conflicts.

Thank you